### PR TITLE
[M1700] CI/CD for new AWS architecture

### PR DESCRIPTION
**Pull request by aspcanada**

### CODE REVIEW

**Comment by saanobhaai** [comment]
[CI/CD for new AWS architecture](https://trello.com/c/fi9URuKV/1700-ci-cd-for-new-aws-architecture)

**Review by saanobhaai — CHANGES_REQUESTED** [review]
### EDIT
Just spoke with Dustin and am sorted out on versions/changelog. I'll create cards (for us) to handle those.

This looks good; I'm going to write out what's different from current circleci workflows, and we can figure out how to handle:

- both dev and prod: 
    - tag and push docker image
    - wait for deployment to complete successfully, then update version

- prod only:
    - update changelog (using trello api)

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: DeployCDKStacks
```
[the remark this answers is not in the document]
Comment (aspcanada) [reply .github/workflows/deploy-dev.yml:1]: Correct. Any updates to `dev` branch will trigger a deploy - same as CircleCI

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (aspcanada) [reply .vscode/settings.json:1]: I usually commit it to [share preferences](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control) for those using vscode.

**Pull request merged**